components/ButtonNavBar/ButtonNavBar.js

```diff
@@ -29,10 +34,19 @@ export default function ButtonNavBar({ id, deleteCard }) {
     });
   }
 
+  function handleToggle(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    toggleMastered(id);
+  }
```

Comment:
you shouldn't have two different toggle functions in two different places (here AND in  _app.js) always choose the best location for the logic and pass the info as props to that location.

Also I think your don't need event.prevent Default here and also not event.stopPropagation, because the buttons are not part of a form :)